.github/dependabot.yml

```diff
@@ -0,0 +1,37 @@
+version: 2
+updates:
+- package-ecosystem: npm
```

Comment:
Verified that only yarn v1 is supported in dependabot, and package-ecosystem in `npm`

https://docs.github.com/en/code-security/supply-chain-security/configuration-options-for-dependency-updates#package-ecosystem